storage/src/lib.rs

```diff
@@ -477,6 +511,9 @@ pub mod initializer {
                 crate::ChainMetaStorage::descriptor(cache),
                 crate::PredecessorStorage::descriptor(cache),
                 crate::BlockAdditionalData::descriptor(&cache),
+                crate::CycleMetaStorage::descriptor(cache),
```

Comment:
@adonagy 
check also `impl TmpStorage {` and add there these storages,
this why interation tests are failing